resty/documents.py

```diff
@@ -57,7 +57,7 @@ def filter(self, name, **kwargs):
             raise ValueError
 
         filter_uri = self.meta.filters[name]
-        for key, value in kwargs.iteritems():
+        for key, value in kwargs.items():
```

Comment:
This could cause some performance issues with large dictionaries in 2.7 as it returns a list of (key, value) pairs instead of an iterator. If we don't expect a lot of data here then it's fine.